src/ngraph/runtime/interpreter/int_executable.cpp

```diff
@@ -100,38 +98,77 @@ runtime::interpreter::INTExecutable::INTExecutable(const std::string& model_stri
 
 bool runtime::interpreter::INTExecutable::call(const vector<shared_ptr<runtime::Tensor>>& outputs,
                                                const vector<shared_ptr<runtime::Tensor>>& inputs)
+{
+    HostTensor::HostEvaluatorTensorVector tensor_outputs;
```

Comment:
Looks like you really just want HostTensor to be a HostEvaluatorTensor. Can we do that?

Reply:
I agree; with some changes to Constant I think I can. And that solves the dynamic_call issue as well.